src/fides/service/messaging/aws_ses_service.py

```diff
@@ -127,13 +129,17 @@ def send_email(
     ) -> None:
         """
         Send an email using AWS SES.
-        Both the from_email and domain set in the messaging config must be verified in SES.
+        Either `email_from` or `domain` must be verified in SES.
         """
         self.validate_email_and_domain_status()
         ses_client = self.get_ses_client()
 
+        from_address = self.messaging_config_details.email_from
+        if not from_address:
+            from_address = f"noreply@{self.messaging_config_details.domain}"
```

Comment:
may just be worth noting in a code comment here the reasoning you described in the PR comment 👍